tools/verify_examples.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+
```

Comment:
Add a short docstring to describe how this script can be used, such as
```
# in the pyscf directory 
./tools/verify_examples.py examples 
./tools/verify_examples.py examples/gto -j 8
```

Additionally, consider to cite this PR in the docstring.

Reply:
Added docstring giving example usage and reference to this PR.